src/accessControl/acp/index.js

```diff
@@ -385,14 +417,92 @@ export default class AcpAccessControlStrategy {
     this.#fetch = fetch;
   }
 
+  getInheritedPoliciesUrlsForResource() {
+    // get all policies that apply to this resource
+    const policies = acpv2.getPolicyUrlAll(this.#originalWithAcr);
+    // filter out owner and own resource policies
+    const namedPolicyUrls = namedPolicies.map(({ name }) => {
+      const containerUrl = getPolicyResourceUrl(
+        this.#originalWithAcr,
+        this.#policiesContainerUrl,
+        name
+      );
+      return getNamedPolicyUrl(containerUrl, name);
+    });
+    const inheritedPoliciesUrls = policies.filter(
+      (policyUrl) =>
+        !namedPolicyUrls.includes(policyUrl) && !policyUrl.includes("#Owner")
```

Comment:
This smells a bit hard-coded, and could potensially lead to problems, e.g. what if someone else creates a policy that starts with `Owner`? It's an edge-case, and probably not going to happen, but is there another way of inferring (and consequently exclude) Owner policies?

Reply:
Also, "#Owner" should be a constant.

Reply:
After a discussion with @megoth where we considered different options, we decided to go with filtering based on the policies container url. If/when we switch from `pb_policies` to `policies`, the owner will no longer be filtered out, so we have updated the corresponding ticket so we can keep this in mind when making that change.